RebelAllianceBank/Classes/ISK.cs

```diff
@@ -3,11 +3,15 @@ namespace RebelAllianceBank.Classes
 {
     public class ISK : IBankAccount
     {
-        public string AccountName { get; set; }
-        public decimal Balance { get; set; }
-        public string AccountCurrency { get; set; }
-        public decimal IntrestRate { get; set; }
+        public int ID { get; set; }
+        public string UserId { get; set; }
+        public int AccountType { get; set; }
```

Comment:
Why is it int and not string?

Reply:
If you meant the UserId, it's because of it's a reference to the IUser PersonalNum